### ISSUE TITLE
OS X Mavericks: Syntax error

### ISSUE THREAD

**Issue by devsli** [bharadwaj-raju/WeatherDesk#5]
# Problem
```shell
% cat ~/.weatherdesk_script.AppleScript
tell application 'Finder'
set desktop picture to POSIX file "/Users/sli/Pictures/FireWatch-WeatherDesk-Pack-master/normal.jpg"
end tell
```
```shell
% osascript ~/.weatherdesk_script.AppleScript
17:18: syntax error: Expected expression, property or key form, etc. but found unknown token. (-2741)
```

# Solution
Replace `'` with `"`
```shell
% cat ~/.weatherdesk_script.AppleScript | sed s/\'/\"/g                   
tell application "Finder"
set desktop picture to POSIX file "/Users/sli/Pictures/FireWatch-WeatherDesk-Pack-master/normal.jpg"
end tell
```
```shell
% cat ~/.weatherdesk_script.AppleScript | sed s/\'/\"/g | osascript
% echo $?
0
```

**Comment by bharadwaj-raju** [comment]
You are late by minutes, friend!

A user just reported this on reddit and it is fixed with commit [47c9717](https://github.com/bharadwaj-raju/WeatherDesk/commit/47c9717ec0ae01ae1cc3563371d31819cdfaed46)

**Comment by devsli** [comment]
Yep, it's working, thanks!
